crates/ra_project_model/src/lib.rs

```diff
@@ -279,11 +279,14 @@ impl ProjectWorkspace {
                             }
                             if tgt.is_proc_macro(&cargo) {
                                 if let Some(proc_macro) = libproc_macro {
-                                    if let Err(_) = crate_graph.add_dep(
```

Comment:
I'd left this in `if let Err(_)` shape, as it better shows that we ignore the error (`_`)